sw/device/silicon_creator/manuf/base/personalization_data.h

```diff
@@ -0,0 +1,80 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_SILICON_CREATOR_MANUF_BASE_PERSONALIZATION_DATA_H_
+#define OPENTITAN_SW_DEVICE_SILICON_CREATOR_MANUF_BASE_PERSONALIZATION_DATA_H_
+#include <stdint.h>
+
+/**
+ * Personalization data is sent between the device and the host during the
+ * device provision. Personalization data is laid out as a sequence of
+ * concatenated LTV objects. The length and type field of the objects are packed
+ * in a 16 bit integer:
+ *  d15                                           d0
+ *  +-------------+--------------------------------+
+ *  | 4 bit type  |   12 bits total  object size   |
```

Comment:
Is there a risk of a certificate ever been bigger than 4kB? If so, this could become a real issue with this encoding.

Reply:
I don't think this is likely at all: will we ever allow certificates spanning more than two INFO pages?

Reply:
4k is just an arbitrary max payload size for all data exported. We can up the size, but realistically we want to keep this as small as possible to maximizing provisioning throughput.